proto/osmosis/incentives/gov.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package osmosis.incentives;
+
+import "gogoproto/gogo.proto";
+import "osmosis/incentives/group.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v19/x/incentives/types";
+
+// CreateGaugeGroupsProposal is a type for creating one or more gauge groups via
+// governance. This is useful for creating gauge groups without having to pay
+// creation fees.
+message CreateGaugeGroupsProposal {
```

Comment:
Whenever we, as a client, interact with `CreateGroup` API, let's refrain from using gauges anywhere since group gauge creation should be abstracted away. As a client, I only know that I'm creating a group to distribute to multiple pools.

Side note: we've been calling `Group`s and group `Gauge`s throughout the module. So let's continue using either `Group` or `GroupGauge` and not `GaugeGroup` for consistency.

```suggestion
// CreateGroupsProposal is a type for creating one or more groups via
// governance. This is useful for creating groups without having to pay
// creation fees.
message CreateGroupsProposal {
```